eucaconsole/views/buckets.py

```
@@ -756,6 +806,15 @@ def bucket_update_versioning(self):
             self.request.error_messages = self.versioning_form.get_errors_list()
         return self.render_dict
 
+    def get_controller_options_json(self):
+        return BaseView.escape_json(json.dumps({
+            'bucket_name': self.bucket.name,
+            'cors_config_xml': self.cors_configuration_xml,
+            'has_cors_config': bool(self.cors_configuration_xml),
```

The corsConfigXML defaults to a sample config when empty, so we'll need the separate hasCorsConfig